File: spring-rest/src/main/java/it/contrader/dto/DettagliCloneDTO.java
package it.contrader.dto;

import it.contrader.model.Utente;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DettagliCloneDTO {

    private long UniqueID;

    private String nome;

    private String cognome;

    private String sesso;

    private double altezza;

    private double peso;

    private String nazionalita;

    private Utente utente;

}

File: src/dto/dettagliCloneDTO.ts
import { UtenteDTO } from "./utenteDTO";

export class DettagliCloneDTO{
    id: number | undefined;
    utenteDTO: UtenteDTO | undefined;
    nome: string | undefined;
    cognome: string | undefined;
    sesso: string | undefined;
    altezza: number | undefined;
    peso: number | undefined;
    nazionalita: string | undefined;
}